internal/openshift/user_idler_map_test.go

```diff
@@ -50,3 +50,19 @@ func Test_len_mutli(t *testing.T) {
 	wg.Wait()
 	assert.Equal(t, n, m.Len(), "Len should return number of items")
 }
+
+// Checking store and delete
+func TestUserIdlerMap_StoreDelete(t *testing.T) {
```

Comment:
could you please try exercising multiple writes and reads as well?

Reply:
OK. I interpreted that as multiple reads and writes in a single goroutine. Would write that.